src/ui/templates/components/fieldset-legend.njk

```diff
@@ -10,7 +10,7 @@
     </legend>
 
     {% else %}
-    <legend class="govuk-label--m" id="{{ id }}-legend" data-cy="{{ id }}-legend">{{ legendText }}</legend>
+    <span class="govuk-label--m" id="{{ id }}-legend" data-cy="{{ id }}-legend">{{ legendText }}</span>
```

Comment:
could we pass the class from constants?

Reply:
Can we do as a tech improvement? We have over 20 instances of "govuk-label--X" that could come from constants.